docs/content/csi_rn_changelog_1.6.0.md

```diff
@@ -0,0 +1,3 @@
+# 1.6.0 (June 2021)
+
+IBM® block storage CSI driver 1.6.0 adds additional support for Kubernetes 1.21 and Red Hat® OpenShift® 4.8 with x86 architecture.
```

Comment:
do we plan to support non-x86 archs for OCP 4.8?

Reply:
I think [we have](https://www.ibm.com/docs/en/stg-block-csi-driver/1.5.0?topic=requirements-supported-orchestration-platforms)

Reply:
Every release OCP version X.Y is supported on x86, Z and P. K8S version W.R is supported only on x86.

Reply:
I updated the wording per DOC-263087